src/feed.ts

```diff
@@ -41,6 +42,13 @@ export class Feed {
    */
   public addExtension = (extension: Extension) => this.extensions.push(extension);
 
+  /**
+   * Adds an extra custom item
+   * @param key
+   * @param value
+   */
+  public addExtra = (key: string, value: ExtraItem) => this.extra[key] = value;
```

Comment:
`ExtraItem` is just a `xml-js` object. Each user needs to make sure that the object conforms to `xml-js` specification.

Reply:
But I'm not sure how I should handle these extra items in the json format.

Reply:
maybe we can simply ignore extra items when generating JSON? Just like extensions in atom1/rss2 format?